### ISSUE TITLE
CTD at the beginning of my turn (random turn each playthrough) 

### ISSUE THREAD

**Issue by hawkseye17** [LoneGazebo/Community-Patch-DLL#6728]
_1. Mod version (i.e Date - 4/17):_
6-19-2

_2. Mod list (if using Vox Populi only, leave blank):_

![Screenshot (43)](https://user-images.githubusercontent.com/67494452/85914670-467ef600-b80e-11ea-82a5-25914478c84d.png)
![Screenshot (44)](https://user-images.githubusercontent.com/67494452/85914665-43840580-b80e-11ea-97a2-3a9b1d3e6398.png)
![Screenshot (45)](https://user-images.githubusercontent.com/67494452/85914666-441c9c00-b80e-11ea-8a65-fe50cf1938f3.png)
![Screenshot (46)](https://user-images.githubusercontent.com/67494452/85914667-44b53280-b80e-11ea-8730-aed234576a6a.png)
![Screenshot (47)](https://user-images.githubusercontent.com/67494452/85914669-45e65f80-b80e-11ea-9156-2cad5e1b11c8.png)

I don't know where to get a mod list to copy-paste so I just took screenshots of the mod list in-game 

_3. Error description:_ 
After a change the turn to to turn 247, the game crashes when my turn comes up, right after the Barbarians end their turn. Keeps happening regardless of which save I use. During the AI turns, I agreed to Friendship declarations with Bismarck and Isabella and took a Research Agreement from Napoleon. During the Barbarian turn, an explorer got bombarded by crossbowmen but that's it. 

_4. Steps to reproduce (optional):_

Just click Next turn and wait for Player turn to a come up
---------------------------
Supporting information: 

I started in the Renaissance Era, but it was really ~62-65 turns in

[Crash Report Supporting information.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/4839977/Crash.Report.Supporting.information.zip)

I tried playing again starting a new game with the same map, AI, settings, etc. And now the game crashes after turn 250 when my turn began on turn 251. This time I did not do any diplomacy on that turn and none of my units got attacked. I also played this time without some mods that could cause compatibility issues 

[Vox Populi Supporting Information 2.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/4841045/Vox.Populi.Supporting.Information.2.zip)

I tried again playing with everything the same as the above except I disabled "Civ-Specific Events". The same crash happened but much earlier after 2 turns playing. Here's the report

[Vox Populi Supporting Information 3.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/4841510/Vox.Populi.Supporting.Information.3.zip)

I'm getting desperate now. I tried playing on a different map with different characters, less mods, and different settings. I started on the Medieval Era this time and the crash still happened on a fixed turn (different than the ones before). The one thing that does not change is that it always happens right after the Barbarian turn ends and my turn starts. I thought it was the event system so I disabled them in the menu before starting the playthrough but that seems to not be the problem. 

[Vox Populi Crash Supporting Information 4.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/4842564/Vox.Populi.Crash.Supporting.Information.4.zip)

I tried another playthrough with even less mods and it happened again. Though something strange happened this time. The game crashed on the start of turn 141 (if I loaded turn 141 from the autosaves, it immediately crashes). I tried doing absolutely nothing after turn 138 and the game did not crash.

I loaded again the 138 save and this time repeated all my moves I did before turn 141 and the game crashed.

What I did before the ctd was:

repair two pillaged improvements

set my explorers to automatically explore

move a swordsman and Chu Ko Nu to attack a barbarian outpost

**Comment by Vyyt02** [comment]
This is a huge number of mods. I would not be surprised if it was caused by a mod conflict or a memory overflow. Also, if I remember correctly, the cities can work 5 tiles mod is very resource- heavy.

**Comment by hawkseye17** [comment]
I tried multiple different times with the same settings and the turn that the crash happened was different each time. The only thing that was constant was that it was at the beginning of my turn. In fact one time it crashed 2 turns in.

**Comment by RecursiveVision** [comment]
@hawkseye17 With that kind of a mod list debugging is more or less impossible. Cities working 5 tiles murders memory and performance, too.

**Comment by hawkseye17** [comment]
@RecursiveVision , I tried another playthrough with even less mods and it happened again. Though something strange happened this time. The game crashed on the start of turn 140 (if I loaded turn 140 from the autosaves, it immediately crashes). I tried doing absolutely nothing after turn 138 and the game did not crash. 

I loaded again the 138 save and this time repeated all my moves I did before turn 140 and the game crashed. 

What I did before the ctd was: 

- repair two pillaged improvements

- set my explorers to automatically explore

- move a swordsman and Chu Ko Nu to attack a barbarian outpost

[Vox Populi Crash Supporting Information 5.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/4842742/Vox.Populi.Crash.Supporting.Information.5.zip)
